websites/B/Backloggd/presence.ts

```diff
@@ -0,0 +1,67 @@
+const presence = new Presence({
+		clientId: "1003092862285651968",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/SUZJZEX.png",
+		},
+		{ pathname } = document.location;
+	if (pathname === "/") presenceData.details = "Viewing the home page";
+	else if (pathname.startsWith("/games/")) {
+		if (pathname.includes("/lib/"))
+			presenceData.details = "Viewing popular games";
+		else {
+			presenceData.details = `Viewing ${document
+				.querySelector("#game-profile h1")
+				.textContent.trim()}`;
+			presenceData.state = `Made by ${document
+				.querySelectorAll("#game-profile #game-body .sub-title a")[1]
+				.textContent.trim()}`;
+			presenceData.largeImageKey = `${
+				document.querySelector<HTMLImageElement>(
+					".game-cover .overflow-wrapper >img"
+				).src
+			}`;
+		}
+	} else if (pathname.includes("/search/")) {
+		presenceData.details = "Searching 🔎";
```

Comment:
Upload the content of [this](https://github.com/PreMiD/Presences/tree/main/.resources) to [the portal](https://discord.com/developers/applications) at rich presence of the current application
```suggestion
		presenceData.details = "Searching";
		presenceData.smallImageKey = "search";
```

Reply:
Thanks but no thanks, Encrypted prefers Imgur Links. Pretty sure emojis are allowed

Reply:
I see imgur links is also fine but you need that specific icon from the folder with images.